package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
I believe the maintainer of that library has to either remove the dependency, or move it in their package.json